### PR TITLE
Update endpoints

### LINKED ISSUE #2: Use domain name instead of IP addresses
The [default options](https://github.com/TDAF/passport-taccounts-oauth2/blob/master/lib/index.js#L73) should point to the public domain name, not private IP addresses.

### PR DESCRIPTION
**Pull request by jmendiara**
TODO: Change Authserver URLs to HTTPs. Fixes #2

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -70,9 +70,9 @@ function TAccountsStrategy(options, verify) {
     Authorization: 'Basic ' + new Buffer(options.clientID + ':' + options.clientSecret).toString('base64')
   };
```
Comment (labajo) [inline lib/index.js:71]: Increment package.json version.

**Comment by labajo** [comment]
LGTM

File: lib/index.js (hunk 1 above)
Comment (jmendiara) [inline lib/index.js:71]: This is not a bump version commit

**Comment by jlorgal** [comment]
LGTM

**Pull request merged**